components/script/canvas_state.rs

```diff
@@ -199,6 +203,10 @@ impl CanvasState {
         &self.ipc_renderer
     }
 
+    pub(crate) fn get_image_key(&self) -> ImageKey {
```

Comment:
Rust typically prefers to omit `get_` in getters.

```suggestion
    pub(crate) fn image_key(&self) -> ImageKey {
```